blocks/bricks/base.py

```diff
@@ -210,12 +210,14 @@ def __getattr__(self, name):
         if 'properties' in self.__dict__ and name in self.properties:
             return property(create_unbound_method(
                 getattr(self, self.properties[name]), self.brick))
-        raise AttributeError
+        raise AttributeError("Cannot find '%s' property in brick %s" %
+                             (name, str(self)))
 
     def __setattr__(self, name, value):
         # Mimic behavior of read-only properties
         if 'properties' in self.__dict__ and name in self.properties:
-            raise AttributeError("can't set attribute")
+            raise AttributeError("Can't set attribute '%s' in brick %s" %
```

Comment:
This error message does not tell the whole story, I believe. The idea here is that `brick.apply.inputs = ['blabla']` is not supported, that is that application method properties are read-only. A good error message should explain this to user.